service/switchboardapp/src/main/java/com/switchboard/app/exceptions/ExceptionResponse.java

```diff
@@ -0,0 +1,25 @@
+package com.switchboard.app.exceptions;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+
+import java.util.Arrays;
+import java.util.List;
+
+@AllArgsConstructor
+@Getter
+public class ExceptionResponse {
+
+    public HttpStatus status;
+    public String message;
+    public List<String> errors;
+
+    public ExceptionResponse(HttpStatus status, String message, String error) {
+        super();
+        this.status = status;
+        this.message = message;
+        errors = Arrays.asList(error);
```

Comment:
Th `Arrays.asList` doesn't seem to be doing much here. When applied on a string, `Arrays.asList` will return a list with a single element which is the string passed. We should change the `error` method parameter to either an array, a String or use varargs.

Reply:
Changed to string for now